apps/docs/content/guides/functions/examples/sentry-monitoring.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Monitoring with Sentry'
+description: 'Monitor Edge Functions with the Sentry Deno SDK.'
+---
+
+Add the [Sentry Deno SDK](https://docs.sentry.io/platforms/javascript/guides/deno/) to your Supabase Edge Functions to easily track exceptions and get notified of errors or performance issues.
+
+### Prerequisites
+
+- [Create a Sentry account](https://sentry.io/signup/).
+- Make sure you have the latest version of the [Supabase CLI](https://supabase.com/docs/guides/cli#installation) installed.
+
+### 1. Create Supabase function
+
+Create a new function locally:
+
+```bash
+supabase functions new sentryfied
+```
+
+### 2. Add the Sentry Deno SDK
+
+Handle exceptions within your function and send them to Sentry.
+
+```tsx
+import * as Sentry from 'https://deno.land/x/sentry/index.mjs'
+
+Sentry.init({
+  dsn: SENTRY_DSN,
```

Comment:
It's not obvious where `SENTRY_DSN` is coming from, so the example wont work as copy-paste.